data/remnant ships.txt

```diff
@@ -414,6 +414,66 @@ ship "Pelican"
 
 
 
+ship "Penguin"
```

Comment:
So despite being a similar size to the Gascraft (and the Penguin is actually slightly slimmer), the Penguin has: better shields, more passenger space, more cargo, less mass, less drag, more outfit space, more engine space, weapon space w/ a gun and a turret hardpoint, better shield generation for the same energy, better hull repair for the same energy, better outfit scan (but 2 worse tac scan), a cloak, and in-built reverse thrusters. I get that it's a newer ship, but that all seems like a bit much of an upgrade to me.

I think the only things I'd say should be changed are:
* Less passenger space. Seems like it'd be pretty cramped with its current bunks.
* No weapons. With less drag, better engines, and a cloak, such a ship could be even more interesting to use in a cat-and-mouse scenario similar to the Gascraft vs Archon scenario. Such a ship shouldn't exactly be designed to fight, so why waste space on it?
* Why in-built reverse engines?
* Up the shield and hull energy a bit to scale with the generation going up.

Reply:
In my (very rough approximation), I consider the gascraft to be one of the oldest of Remnant ships - One sufficient old that it is likely one of the original science vessels that has been reinforced to deal with gasgiants, and more recently replacing its power system to remove the supercapacitors. 

So when comparing the two, keep in mind that you are comparing the Penguin with a ship that predates the Remnant even having access to the materials technology that makes their structures and ships distinctive. And this one is the first prototype of what appears to be an entirely new generation of Remnant ships. At least, this is may be the first ship that the player sees the Remnant actually complete the development of. All the other ships (including the recently PRd ones), from the Starling to the Ibis, are all ships that currently exist when the player finds the Remnant.

That being said, I agree. I've dropped the bunks down to 3, and increased the shield and hull regeneration energy.

The main reason for the gun port is to give people the option of installing an actual reverse thruster. If used for anything else, I think this would probably fall into the category of the "As useless as a gun on a shuttle" thing. Maybe not *quite* as useless, but close.

The small amount of built-in reverse thrusters reflects two parts of the design:
1) The ship is actually capable of movement on its own that isn't tied to the main thrusters. This is a good way to represent it.
2) It is a research vessel designed primarily for conducting research in-atmosphere around large living creatures. Most biologists agree that if one wants to study a creature, it is best to be able to approach them and move around without any sudden motions (like swinging around to counter-thrust). Thus it has a small amount of reverse thrust, enough that it could slowly approach a void sprite and bring itself to a stop without swinging around.